Dockerfile

```diff
@@ -1,11 +1,8 @@
 FROM alpine:latest
+USER root
```

Comment:
Will this work with psp enabled https://github.com/rancher/charts/blob/dev-v2.6/packages/rancher-istio/charts/templates/istio-install-job.yaml#L45-L49 ?

Reply:
This needed to be removed, it was left behind due to some different stuff I was trying in the dockerfile and is no longer needed.